auth/client/src/test/java/org/wildfly/security/auth/client/DefaultSSLContextProviderCanDelegateWhenConfigIsMissingTestCase.java

```diff
@@ -0,0 +1,28 @@
+package org.wildfly.security.auth.client;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import javax.net.ssl.SSLContext;
+import java.security.NoSuchAlgorithmException;
+import java.security.Security;
+
+public class DefaultSSLContextProviderCanDelegateWhenConfigIsMissingTestCase {
+
+    @Test
+    public void testDefaultSSLContextProviderWillDelegateIfNoConfigured() {
+        Security.insertProviderAt(new ElytronClientDefaultSSLContextProvider(), 1);
+        Assert.assertEquals(ElytronClientDefaultSSLContextProvider.class.getSimpleName(), Security.getProvider("ElytronClientDefaultSSLContextProvider").getName());
```

Comment:
IIUC, here you mean to check that the inserted provider is the `ElytronClientDefaultSSLContextProvider`, which seems right to me but in the previous test class you just checked for the named one not to be null. Can we have a single way to assess that the provider registration was successful and consistent?

Reply:
@fabiobrz If the provider is not null it is certain that this equality is true, so I left the assertNotNull check.  And I changed this line to be consistent everywhere.

Reply:
Great, thanks!